package-lock.json

```diff
@@ -5804,15 +5888,6 @@
       "resolved": "https://registry.npmjs.org/safer-buffer/-/safer-buffer-2.1.2.tgz",
       "integrity": "sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg=="
     },
-    "sandboxed-module": {
```

Comment:
This package is listed with a package version of `""` and somehow went missing during the `npm install` stage. This sounds like an npm-bug to me.

The `chai` package was removed as well.

Would you mind specifying the versions from the package-lock in the package.json and amend that with the output of `dev-env$ make spelling/npm_install`?

```
"chai": "4.2.0",
"sandboxed-module": "2.0.4",
```

Reply:
Done in https://github.com/overleaf/spelling/pull/67/commits/a7c2cc6dcab0a4308baf55722fc88671d59a65fd